docs/snippets/jmespath/powertoolsBase64JmespathPayload.json

```diff
@@ -0,0 +1,3 @@
+{
+  "payload": "eyJ1c2VyX2lkIjogMTIzLCAicHJvZHVjdF9pZCI6IDEsICJxdWFudGl0eSI6IDIsICJwcmljZSI6IDEwLjQwLCAiY3VycmVuY3kiOiAiVVNEIn0="
```

Comment:
Same for this.

Reply:
I have added it in the sample using the annotation feature, which looks like this:

<img width="863" alt="Screenshot 2024-03-24 at 20 26 45" src="https://github.com/aws-powertools/powertools-lambda-typescript/assets/7353869/5be5c3f2-459a-458c-a4cf-6a07229aaad1">